docs/knowledge/error-handling-unification.md

```
@@ -0,0 +1,223 @@
+
+/// エラーメッセージの二層構造
+type ErrorMessage =
+    { UserMessage: string        // ユーザー向け（日本語、分かりやすい）
+      TechnicalDetails: string   // 開発者向け（英語、詳細）
+      RecoveryHint: string option } // 復旧方法の提案
+```
+
+### Railway Oriented Programming採用
+
+```fsharp
+/// 操作結果のログ出力（統一パターン）
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)